src/main/java/graphql/DirectiveInfo.java

```diff
@@ -0,0 +1,52 @@
+package graphql;
+
+import graphql.schema.GraphQLDirective;
+
+import java.util.LinkedHashMap;
+import java.util.LinkedHashSet;
+import java.util.Map;
+import java.util.Set;
+
+/**
+ * Info on all the directives provided by graphql specification
+ */
+@PublicApi
+public class DirectiveInfo {
```

Comment:
Can we please move this to  `graphql.schema.idl` so its next to `graphql.schema.idl.ScalarInfo` - its pretty much the same idea as a class and hence it should be in a similar place

Reply:
@bbakerman I have overlooked this. thanks.